src/app-layout/__integ__/awsui-applayout.test.ts

```diff
@@ -197,4 +197,15 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as Theme[])('%s', theme
       await expect(page.getNavPosition()).resolves.toEqual(navBefore);
     })
   );
+
+  testIf(theme === 'refresh-toolbar')(
```

Comment:
Can we run it for other designs too?

Reply:
No, VR and classic have an additional gap that prevents the header from becoming sticky immediately. We’d need additional conditions for them, which isn’t the purpose of this test. See the attached video for reference.

https://github.com/user-attachments/assets/6d03be33-cecd-4d2a-9dd8-c27ef7d5265d

Reply:
I think we could still run the test, just have a different assertions for each mode. But up to you